src/showList.js

```diff
@@ -0,0 +1,27 @@
+export default function showList(tasksList) {
+  const listDiv = document.getElementById('list');
+  listDiv.innerHTML = '';
+  for (let index = 0; index < tasksList.length; index += 1) {
+    const task = document.createElement('li');
+    const remove = document.createElement('i');
+    const description = document.createElement('p');
+    if (tasksList[index].completed) {
+      const checked = document.createElement('i');
+      checked.className = 'fas fa-check';
+      description.appendChild(checked);
```

Comment:
- [ ] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidlines.